cmd/clean-cache.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/go-flutter-desktop/hover/internal/enginecache"
+	"github.com/go-flutter-desktop/hover/internal/log"
+)
+
+var cachePath string
+
+func init() {
+	cleanCacheCmd.Flags().StringVar(&cachePath, "cache-path", enginecache.DefaultCachePath(), "The path that hover uses to cache dependencies such as the Flutter engine .so/.dll")
+	rootCmd.AddCommand(cleanCacheCmd)
+}
+
+var cleanCacheCmd = &cobra.Command{
+	Use:   "clean-cache",
+	Short: "Clean the engine cache",
```

Comment:
This sounds like it is a cache that the engine uses.
Maybe rephrase like "Cleanup cached engines" or something like that?